spec/takeaway_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'takeaway'
+
+describe Takeaway do
+let(:menu) { double :menu }
+let(:first_order) {double :first_order }
+subject {described_class.new(menu)}
+
+
+it { is_expected.to respond_to(:pick).with(2).arguments }
+  
+it { is_expected.to respond_to(:check_sum) }
+
+it { is_expected.to respond_to(:basket) }
+
+  describe '#initialize' do
+    it 'creates a menu' do
+    expect(subject.menu).to eq menu
+    end
+  end
+
+ describe '#new_order' do
+   
+   it 'creates a new order' do
+    allow(subject).to receive(:new_order).and_return first_order
+    allow(menu).to receive(:menu).and_return({"Burger" => 5.00, "Pizza" => 4.00, "Fish" => 6.00, "Chicken" => 5.00})
+    allow(menu).to receive(:view).and_return("Burger" => 5.00, "Pizza" => 4.00, "Fish" => 6.00, "Chicken" => 5.00)
```

Comment:
The fact that you have these two lines makes it clear to me that view and menu do not both need to exist